### ISSUE TITLE
Who is involved? Updates and additions

### ISSUE THREAD

**Issue by masinter** [Interlisp/medley#1092]
context; "the call" was a call with educopia (sponsors of the Software Preservation network) and what kind of consulting they could offer.

@rmkaplan via email:
> Coming off of our call, I looked at the Interlisp web page.  I see that you have
> a link to your page.  If there are links (e.g. linked in) for some of the other
> people, should we put them in? Might make us look more serious.
> 
> For me, maybe the wikipedia page.
> https://en.wikipedia.org/wiki/Ronald_Kaplan
@masinter 
> I can add your link, but I was thinking of replacing the whole section with the gloss from the [annual report](https://docs.google.com/document/d/1cGBDNMO5yt6ymi7YiCcf6uNn6RBe5PVS7mW4kFnlzNY/edit#heading=h.kly6krobw8zi)
> Section (toward the end): Who are we?
and the "We" comes from participating in mail, github, or community activities.

The problem with a list of people is that it doesn't sustain. -- I put in the list that's there a year ago. As John Cowan said one meeting -- people who want to know can look, and decide themselves.
And open source projects (like XQuartz!, whose financials are managed by SPI) what do they do?

**Comment by masinter** [comment]
I think we've made quite a bit of progress on the original issue. The website has a page --
[Getting Involved](https://interlisp.org/about/getinvolved/) about how newcomers can get involved, which also serves as a categorization of who else can contribute.

Maybe we should point to the Issues that are marked "Help Wanted" ?

The Wiki had a "Guest Book". Do one as a Google Form? Revive the wiki for features that aren't part of the new web site?

We need some better text about the OG's of the project (Ron, Me, Nick, Frank, Herb?)

Everyone on LispCore should have a brief introduction. Do we prefer GitHub profiles?

**Comment by pamoroso** [comment]
Issue #1056 has a possible initial list of "Help Wanted" issues.

I seemed to understand the Wiki was discontinued: is it still used?

As for introductions, GitHub profiles are good starting points as they are easy to customize and update.

**Comment by masinter** [comment]
We moved 95% of the wiki content to the (hugo-baased) website (and much of the material on the old website). but we haven't deleted them, there's just a notice at top that the content is "in transition".
Cleaning this up is a "help wanted" task but hard to delegate completely; what things to keep and what to throw away. For example, are there any old pointers to content we've obsoleted. Without trapping 404's it's hard to know what indirect we should serve.

**Comment by masinter** [comment]
For the BALISP presentation I did add a slide that listed people's names, but (a) I left out some contributors (not on purpose) and I wound up sorting by length of name.
I was thinking of moving the "goals" document into the GitHub Discussion as a parallel to the screenshot pages, to cover individuals, their background, interest in Lisp etc.

Now added to "website" project

**Comment by stumbo** [comment]
Closing this issue.  Recognizing that there is more work to do in compiling the names of individuals associated with Interlisp and Medley I have opened issue #1241 as a place to continue to work that aspect of this issue.

**Comment by masinter** [comment]
I think the credits page really still needs work, and the comments haven't been addressed.
Some important contributors are missing. The contributors list needs a better categorization
"People contributing to the 2020's Medley Interlisp Project" vs "People who helped build Interlisp and Medley" which we are not listing currently, but including those whose listings are in memoriam.

Probably this should incude a discussion of "initials" in functions and who did what, the :EDIT-BY addition, FILECREATED etc, the use of version control at Envos and the PARC->CHM donation sources.

Anyway I am reopening this issue....

**Comment by masinter** [comment]
@pmcjones suggested taking a look at https://multicians.org/multicians.html
as a solution another volunteer project took.

**Comment by pamoroso** [comment]
In the February 19, 2025 external meeting we agreed to address with an issue how to acknowledge other project contributors on the site, for example on the Credits page as @stumbo suggested, or elsewhere such as on GitHub. #1092 and #1427 are related: should we continue the discussion in one of them or open a new issue?

**Comment by masinter** [comment]
I'd suggest using this issue https://github.com/Interlisp/medley/issues/1092 to consolidate issues about the "Credits".  

Proposal:   For 2025 and going forward, we expand "Credits" in the annual report; and we leave the web site Credits to list major contributors 2020-2024.  Each major contributor should have their full name, at least one link to an information page about them (web site, wikipedia page, linkedIn profile ) and a brief summary of what they did, with a link to the issues/prs or files explaining.

The main purpose is to acknowledge our dependence on volunteers, give credit, and encourage future contributions.

**Comment by pamoroso** [comment]
I agree with the proposal. I added to the [2025 annual report outline](https://docs.google.com/document/d/1L9KVZx6BV920wwjWqfNNywTeSq9Xb0r0BRyIV2Bp7hc/edit?usp=sharing) a Credits section under Project and Organization, which I will update over the year.

As for the Credits page of the website, one of the first things to do should probably be to reformat the inline sequence of names as a series of entries in a list or paragraphs where the relevant information can be added (e.g. link, summary of contributions, references to issues/PRs).
